packages/core/src/external/carequality/command/documents/send-ihe-gateway-results.ts

```diff
@@ -0,0 +1,128 @@
+import { Sequelize, QueryTypes } from "sequelize";
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { RaceControl, checkIfRaceIsComplete, controlDuration } from "../../../../util/race-control";
+import { MetriportError } from "../../../../util/error/metriport-error";
+import { initSequelizeForLambda } from "../../../../util/sequelize";
+import { errorToString } from "../../../../util/error/shared";
+
+const REQUEST_ID_COLUMN = "request_id";
```

Comment:
Can we make this so the compiler will complain if we change the column name on the model and forget to update this one here, please?

Or reference them all from the same place. We did this before, don't remember exactly where, but its on the OSS codebase.